app/src/main/java/com/framgia/soundclound/screen/detailgenre/GenreDetailViewModel.java

```diff
@@ -26,23 +30,39 @@
         MoreInfoClickListener {
 
     private GenreDetailAdapter mGenreDetailAdapter;
-    private String mGenre;
+    private Genre mGenre;
     private Context mContext;
     private TrackRepository mTrackRepository;
+    public ObservableField<EndSrcollListenner> mEndSrcollListenner = new ObservableField<>();
```

Comment:
WARNING: Variable 'mEndSrcollListenner' must be private and have accessor methods.